src/modules/Dashboard/components/CurrentImageTab/CurrentImageTab.tsx

```diff
@@ -68,77 +61,70 @@ export default class CurrentImageTab extends PureComponent<IProps, IState> {
 			},
 		} = this.props;
 
+		const commonTab = {
+			title: <Trans i18nKey='common.Common'/>,
+			content: (
+				<div className={styles.header}>
+					<img className={styles.header_img} src={thumb} />
+					<div className={styles.header_createdAt}>
+						<span>Created at: </span>
+						<strong className={styles.header_createdAt_date}>21.09.2018</strong>
+					</div>
+				</div>
+			),
+		};
+		const packagesTab = {
+			title: <Trans i18nKey='common.Packages'/>,
+			content: (
+				<div className={styles.dropDownWrapper}>
+					<DropDownTab
+						title={<Trans i18nKey='Packages.Base'/>}
+						value={basePackages.length}
+						isOpened={!!openedDropdownTabs.base}
+						onClick={() => this.onToggleDropdownTabs('base')}
+					>
+						{
+							basePackages.map((pack: any, index: any) => (
+								<div key={index}>{pack.package}</div>
+							))
+						}
+					</DropDownTab>
+					<DropDownTab
+						title={<Trans i18nKey='Packages.Dependencies'/>}
+						value={depPackages.length}
+						isOpened={!!openedDropdownTabs.dependencies}
+						onClick={() => this.onToggleDropdownTabs('dependencies')}
+					>
+						{
+							depPackages.map((pack: any, index: any) => (
+								<div key={index}>{pack.package}</div>
+							))
+						}
+					</DropDownTab>
+					<DropDownTab
+						title={<Trans i18nKey='Packages.Added'/>}
+						value={addedPackages.length}
+						isOpened={!!openedDropdownTabs.added}
+						onClick={() => this.onToggleDropdownTabs('added')}
```

Comment:
or use a carrying method, and use a one ref on your function, like:

in render:

const myFunc = () => type => this.onToggleDropdownTabs(type)

myFunc()('added') -> that way create a ONE reference on each render

Reply:
Your ways:
![image of error](https://cdn.discordapp.com/attachments/418795190061629441/545992062001217536/unknown.png)